pkg/api/types.go

```diff
@@ -836,9 +942,13 @@ type ResourceName string
 
 const (
 	// CPU, in cores. (500m = .5 cores)
-	ResourceCPU ResourceName = "cpu"
+	ResourceCPU			ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
-	ResourceMemory ResourceName = "memory"
+	ResourceMemory 		ResourceName = "memory"
+	// volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 2014 * 2014)
+	ResourceSize		ResourceName = "size"
+	ResourceIOPS    	ResourceName = "iops"
```

Comment:
These are gone, but the map of many ResourceRequirements still exists, so we can go from 1 to many.  Size was renamed to "ResourceStorage", as suggested.